### ISSUE TITLE
Audit local link references for 'Page not Found' errors

### ISSUE THREAD

**Issue by alfarok** [w3f/polkadot-wiki#3449]
This issue builds off #3136 which was recently closed.  That task involved resolving all the external link urls but did not include references to other local markdown.  While scanning the console output for build warnings and errors I noticed the following:

```
[INFO] Docusaurus found broken links!
Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.
Exhaustive list of all broken links found:
- On source page path = /docs/build-wallets:
   -> linking to learn-treasury.md (resolved as: /docs/learn-treasury.md)

```

I need to look more into how this was generated but it points to at least 1 real broken link:
https://wiki.polkadot.network/docs/learn-treasury.md

If we can validate that the Docusaurus scan is checking all the local links, the resolution to this issue is simply fixing the one path outlined above.  If other local links are determined to be broken it may be worth looking into scanning the repo with another strategy or investigating how Docusaurus is performing the check.

**Comment by alfarok** [comment]
These errors are generate by Docusaurus due to [this parameter](https://github.com/w3f/polkadot-wiki/blob/master/polkadot-wiki/docusaurus.config.js#L46) in the `docusaurus.config.js` file.
```
[INFO] Docs markdown link couldn't be resolved: (learn-treasury.md) in /home/alfarok/Documents/Git/polkadot-wiki-main/docs/build/build-wallets.md for version current
[INFO] Docs markdown link couldn't be resolved: (learn-treasury.md) in /home/alfarok/Documents/Git/polkadot-wiki-main/docs/build/build-wallets.md for version current
[INFO] Docs markdown link couldn't be resolved: (kusama-endpoints.md) in /home/alfarok/Documents/Git/polkadot-wiki-main/docs/general/kusama/kusama-getting-started.md for version current
[INFO] Docs markdown link couldn't be resolved: (kusama-endpoints.md) in /home/alfarok/Documents/Git/polkadot-wiki-main/docs/general/kusama/kusama-getting-started.md for version current
[INFO] Docs markdown link couldn't be resolved: (learn-treasury.md) in /home/alfarok/Documents/Git/polkadot-wiki-main/docs/build/mirror-build-wallets.md for version current
[INFO] Docs markdown link couldn't be resolved: (learn-treasury.md) in /home/alfarok/Documents/Git/polkadot-wiki-main/docs/build/mirror-build-wallets.md for version current
```

There are settings for `onBrokenLinks` and `onBrokenMarkdownLinks`.  Only the later is showing any results right now which makes me think we  either fixed all the external links or that built-in check isn't working properly.

**Comment by alfarok** [comment]
addressed in #3480
